sql/search_aggregates_v5.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TEMP FUNCTION
+  udf_mode_last(list ANY TYPE) AS ((
+    SELECT
+      _value
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS
+    _offset
+    GROUP BY
+      _value
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1 ));
+
+WITH
+  summary_addon_version AS (
+  SELECT
+    *,
+    udf_mode_last(ARRAY(
+    SELECT
+      element.version
+    FROM
+      UNNEST(active_addons.list)
+    WHERE
+      element.addon_id = 'followonsearch@mozilla.com')) AS addon_version
+  FROM
+    main_summary_v4
+  ),
+  augmented AS (
+  SELECT
+    s.*,
+    sc.element.source AS source,
+    sc.element.engine AS engine,
+    sc.element.count AS count,
```

Comment:
The `AS` clauses here are redundant, 
```suggestion
    sc.element.count,
```